### PR TITLE
Correct Kotlin Flow/Flux/Publisher convertors

### PR DESCRIPTION
**Pull request by dstepanov**
Somehow it doesn't work to have a Java class and have it as a bean in the Koltin project. I have separated two converters that are registered using the service loader, in case one of the classes is not present it should reject the converter.

### CODE REVIEW

**Comment by dstepanov** [comment]
Looks like they need to be beans with the requires annotation

File: kotlin-runtime/build.gradle [hunk 1]
```diff
@@ -10,7 +10,8 @@ dependencies {
 
     compileOnly libs.kotlinx.coroutines.core
     compileOnly libs.kotlinx.coroutines.reactive
-    compileOnly(libs.graal.svm)
+    compileOnly libs.kotlinx.coroutines.reactor
```
Comment (yawkat) [inline kotlin-runtime/build.gradle:13]: is this correct? i think asFlow would be missing without this dependency, but the converter can still be called if kotlinx-coroutines and reactor-core are on the classpath

**Review by yawkat — APPROVED** [review]

File: kotlin-runtime/build.gradle (hunk 1 above)
Comment (dstepanov) [reply kotlin-runtime/build.gradle:13]: Dependencies are required to be added by the user

Comment (yawkat) [reply kotlin-runtime/build.gradle:13]: ok

**Pull request merged**